admin_manual/configuration_user/user_auth_ldap.rst

```diff
@@ -466,6 +467,84 @@ In migrated Nextcloud installations the old behavior still applies, which is usi
 
   sudo -u www-data php occ config:app:set user_ldap enforce_home_folder_naming_rule --value=1
 
+"$home" Placeholder Field:
+  $home in an external storage configuration will be replaced with the value of the specified attribute
+
+.. _LDAP_User_Profile_Attributes:
+
+User Profile attributes
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. figure:: ../images/ldap-advanced-4-attributes.png
+   :alt: User Profile Attributes.
+
+After configuring those attributes, the User Profile data will be overwritten with the according data from LDAP.
+The data must be formatted like Nextcloud wants it, otherwise it will be constantly changed and rechanged/removed again.
+
+:Please be aware:
+- The user can not change the data in profile, for it will get overwritten almost instantly
+- The user can change the visibility scope in profile
+- The default visibility can be adjusted with setting the ``account_manager.default_property_scope`` array in ``config.php``
+- If multiple attribute values are present, only the first value is used
```

Comment:
You're right and yet In my experience the values were always returned in the order of addition. So, I have choosen not to worry about it, because sorting of values or anything else would burn server ressource. Instead I have modified my schema and set attributes to SINGLE-VALUE.
Also. The eMail is handled likewise. First value gets primary eMail and additional values get additional eMail address. And it seems to be no problem.